tests/test_wps_parameters.py

```diff
@@ -21,3 +21,19 @@ def test_parameters_local(config):
         temp_config.read()
         params = f"config={temp_config.name};"
         run_wps_process(Parameters(), params)
+
+
+@pytest.mark.online
+@pytest.mark.parametrize(
+    ("config"), [resource_filename(__name__, "configs/parameter_https.cfg")],
+)
+def test_parameters_https(config, conftest_make_mock_urls):
+    config_name = os.path.splitext(config)[0]  # Remove .cfg extension
+    with NamedTemporaryFile(
+        suffix=".cfg", prefix=os.path.basename(config_name), mode="w+t"
+    ) as temp_config:  # Avoid permanent replacement of https URLs
+        read_config = open(config, "r")
+        temp_config.writelines(read_config.read())
+        temp_config.read()
```

Comment:
Is there a reason we aren't just directly reading into the `temp_config`?

Reply:
I'm not sure what you mean here.